### PR TITLE
Keyprez bison

### PR DESCRIPTION
**Pull request by csandven**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/keyprez/bison/keymaps/default_6_6/keymap.c [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    *            | LGUI | LAlt | LCTR |LOWER |  BS     /       \Space    |RAISE | RCTR | RAlt | FN   |
+    *            |      |      |      |      |        /         \        |      |      |      |      |
+    *             `----------------------------------'           '------''---------------------------'
+    */
+    [_QWERTY] = LAYOUT_6_6( \
+        KC_GRV,   KC_1,   KC_2,    KC_3,    KC_4,    KC_5,                      KC_6,    KC_7,    KC_8,    KC_9,    KC_0,  KC_DEL, \
+        KC_TAB,   KC_Q,   KC_W,    KC_E,    KC_R,    KC_T,                      KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,  KC_BSPC,\
+        ESC_CTL,  KC_A,   KC_S,    KC_D,    KC_F,    KC_G,                      KC_H,    KC_J,    KC_K,    KC_L, KC_SCLN,  KC_QUOT,\
+        KC_LSFT,  KC_Z,   KC_X,    KC_C,    KC_V,    KC_B, KC_MUTE,     KC_MPLY,KC_N,    KC_M, KC_COMM,  KC_DOT, KC_SLSH,  KC_ENT, \
+                                          KC_LGUI,KC_LALT, LOWER,       KC_BSPC,              KC_SPC,      RAISE, KC_RALT, FN \
```
Comment (fauxpark) [inline keyboards/keyprez/bison/keymaps/default_6_6/keymap.c:70]:
```suggestion
    [_QWERTY] = LAYOUT_6_6(
        KC_GRV,   KC_1,   KC_2,    KC_3,    KC_4,    KC_5,                      KC_6,    KC_7,    KC_8,    KC_9,    KC_0,  KC_DEL,
        KC_TAB,   KC_Q,   KC_W,    KC_E,    KC_R,    KC_T,                      KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,  KC_BSPC,
        ESC_CTL,  KC_A,   KC_S,    KC_D,    KC_F,    KC_G,                      KC_H,    KC_J,    KC_K,    KC_L, KC_SCLN,  KC_QUOT,
        KC_LSFT,  KC_Z,   KC_X,    KC_C,    KC_V,    KC_B, KC_MUTE,     KC_MPLY,KC_N,    KC_M, KC_COMM,  KC_DOT, KC_SLSH,  KC_ENT,
                                          KC_LGUI,KC_LALT, LOWER,       KC_BSPC,              KC_SPC,      RAISE, KC_RALT, FN
```

File: keyboards/keyprez/bison/keymaps/default_6_6/keymap.c [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+    *            | LGUI | LAlt | LCTR |LOWER |  BS     /       \Space \  |RAISE | RCTR | RAlt | FN   |
+    *            |      |      |      |      |        /         \      \ |      |      |      |      |
+    *             `----------------------------------'           '------''---------------------------'
+    */
+    [_COLEMAK] = LAYOUT_6_6( \
+        KC_GRV,   KC_1,   KC_2,    KC_3,    KC_4,    KC_5,                      KC_6,    KC_7,    KC_8,    KC_9,    KC_0,  KC_DEL,  \
+        KC_TAB,   KC_Q,   KC_W,    KC_F,    KC_P,    KC_G,                      KC_J,    KC_L,    KC_U,    KC_Y, KC_SCLN,  KC_BSPC, \
+        ESC_CTL,  KC_A,   KC_R,    KC_S,    KC_T,    KC_D,                      KC_H,    KC_N,    KC_E,    KC_I,    KC_O,  KC_QUOT, \
+        KC_LSFT,  KC_Z,   KC_X,    KC_C,    KC_V,    KC_B, KC_MUTE,     XXXXXXX,KC_K,    KC_M, KC_COMM,  KC_DOT, KC_SLSH,  KC_ENT,  \
+                                          KC_LGUI,KC_LALT, LOWER,       KC_BSPC,              KC_SPC,      RAISE, KC_RALT, FN \
```
Comment (fauxpark) [inline keyboards/keyprez/bison/keymaps/default_6_6/keymap.c:93]:
```suggestion
    [_COLEMAK] = LAYOUT_6_6(
        KC_GRV,   KC_1,   KC_2,    KC_3,    KC_4,    KC_5,                      KC_6,    KC_7,    KC_8,    KC_9,    KC_0,  KC_DEL,
        KC_TAB,   KC_Q,   KC_W,    KC_F,    KC_P,    KC_G,                      KC_J,    KC_L,    KC_U,    KC_Y, KC_SCLN,  KC_BSPC,
        ESC_CTL,  KC_A,   KC_R,    KC_S,    KC_T,    KC_D,                      KC_H,    KC_N,    KC_E,    KC_I,    KC_O,  KC_QUOT,
        KC_LSFT,  KC_Z,   KC_X,    KC_C,    KC_V,    KC_B, KC_MUTE,     XXXXXXX,KC_K,    KC_M, KC_COMM,  KC_DOT, KC_SLSH,  KC_ENT,
                                          KC_LGUI,KC_LALT, LOWER,       KC_BSPC,              KC_SPC,      RAISE, KC_RALT, FN
```

File: keyboards/keyprez/bison/keymaps/default_6_6/keymap.c [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+    *            |      |      |      |      |         /       \      \  |      |      |      |      |
+    *            |      |      |      |      |        /         \      \ |      |      |      |      |
+    *             `----------------------------------'           '------''---------------------------'
+    */
+    [_LOWER] = LAYOUT_6_6( \
+        _______, _______, _______, _______, _______, _______,                      _______, _______, _______, _______, _______, _______, \
+        _______,   KC_F1,   KC_F2,   KC_F3,  KC_F4,  _______,                      _______, _______, _______, KC_LBRC, KC_RBRC, KC_BSLS, \
+        _______,   KC_F5,   KC_F6,   KC_F7,  KC_F8,  _______,                      KC_LEFT, KC_DOWN, KC_UP,  KC_RIGHT, _______, _______, \
+        _______,   KC_F9,  KC_F10,  KC_F11, KC_F12,  _______, _______,     _______,KC_HOME, _______, _______,  KC_END, _______, _______, \
+                                          _______,_______, _______,       _______,              _______,      _______, _______, _______ \
```
Comment (fauxpark) [inline keyboards/keyprez/bison/keymaps/default_6_6/keymap.c:116]:
```suggestion
    [_LOWER] = LAYOUT_6_6(
        _______, _______, _______, _______, _______, _______,                      _______, _______, _______, _______, _______, _______,
        _______,   KC_F1,   KC_F2,   KC_F3,  KC_F4,  _______,                      _______, _______, _______, KC_LBRC, KC_RBRC, KC_BSLS,
        _______,   KC_F5,   KC_F6,   KC_F7,  KC_F8,  _______,                      KC_LEFT, KC_DOWN, KC_UP,  KC_RIGHT, _______, _______,
        _______,   KC_F9,  KC_F10,  KC_F11, KC_F12,  _______, _______,     _______,KC_HOME, _______, _______,  KC_END, _______, _______,
                                          _______,_______, _______,       _______,              _______,      _______, _______, _______
```

File: keyboards/keyprez/bison/keymaps/default_6_6/keymap.c [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+    *            |      |      |      |      |         /       \      \  |      |      |      |      |
+    *            |      |      |      |      |        /         \      \ |      |      |      |      |
+    *             `----------------------------------'           '------''---------------------------'
+    */
+    [_RAISE] = LAYOUT_6_6( \
+        _______, _______, _______, _______, _______, _______,                      _______, _______, _______, _______, _______, _______, \
+        _______,   KC_F1,   KC_F2,   KC_F3,  KC_F4,  _______,                      _______, _______,KC_PRVWD,   KC_UP,KC_NXTWD, _______, \
+        _______,   KC_F5,   KC_F6,   KC_F7,  KC_F8,  _______,                      _______, _______, KC_LEFT, KC_DOWN,KC_RIGHT, _______, \
+        _______,   KC_F9,  KC_F10,  KC_F11, KC_F12,  _______, _______,     _______,_______, _______, _______, _______, _______, _______, \
+                                          _______,_______, _______,       _______,              _______,      _______, _______, _______ \
```
Comment (fauxpark) [inline keyboards/keyprez/bison/keymaps/default_6_6/keymap.c:139]:
```suggestion
    [_RAISE] = LAYOUT_6_6(
        _______, _______, _______, _______, _______, _______,                      _______, _______, _______, _______, _______, _______,
        _______,   KC_F1,   KC_F2,   KC_F3,  KC_F4,  _______,                      _______, _______,KC_PRVWD,   KC_UP,KC_NXTWD, _______,
        _______,   KC_F5,   KC_F6,   KC_F7,  KC_F8,  _______,                      _______, _______, KC_LEFT, KC_DOWN,KC_RIGHT, _______,
        _______,   KC_F9,  KC_F10,  KC_F11, KC_F12,  _______, _______,     _______,_______, _______, _______, _______, _______, _______,
                                          _______,_______, _______,       _______,              _______,      _______, _______, _______
```

File: keyboards/keyprez/bison/keymaps/default_6_6/keymap.c [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+    *            |      |      |      |      |         /       \      \  |      |      |      |      |
+    *            |      |      |      |      |        /         \      \ |      |      |      |      |
+    *             `----------------------------------'           '------''---------------------------'
+    */
+    [_HRM] = LAYOUT_6_6( \
+        _______, _______, _______, _______,_______, _______,                      _______, _______, _______, _______, _______, _______, \
+        _______, _______, _______, _______,_______, _______,                      _______, _______, _______, _______, _______, _______, \
+        _______,  HRM_LG,  HRM_LA,  HRM_LC, HRM_LS, _______,                      _______,  HRM_RS,  HRM_RC,  HRM_RA,  HRM_RG, _______, \
+        _______, _______, _______, _______,_______, _______, _______,     _______,_______, _______, _______, _______, _______, _______, \
+                                          _______,_______, _______,       _______,              _______,      _______, _______, _______ \
```
Comment (fauxpark) [inline keyboards/keyprez/bison/keymaps/default_6_6/keymap.c:162]:
```suggestion
    [_HRM] = LAYOUT_6_6(
        _______, _______, _______, _______,_______, _______,                      _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______,_______, _______,                      _______, _______, _______, _______, _______, _______,
        _______,  HRM_LG,  HRM_LA,  HRM_LC, HRM_LS, _______,                      _______,  HRM_RS,  HRM_RC,  HRM_RA,  HRM_RG, _______,
        _______, _______, _______, _______,_______, _______, _______,     _______,_______, _______, _______, _______, _______, _______,
                                          _______,_______, _______,       _______,              _______,      _______, _______, _______
```

File: keyboards/keyprez/bison/keymaps/default_6_6/keymap.c [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+    *            |      |      |      |      |         /       \      \  |      |      |      |      |
+    *            |      |      |      |      |        /         \      \ |      |      |      |      |
+    *             `----------------------------------'           '------''---------------------------'
+    */
+    [_FN] = LAYOUT_6_6( \
+        _______, _______, _______, _______, _______, _______,                      _______, _______, _______, _______, _______, _______, \
+        _______,KC_QWRTY, _______, _______, _______, _______,                      _______, _______, _______, _______, _______, _______, \
+        _______, _______, _______, _______, _______, _______,                       KC_HRM, _______, _______, _______, _______, _______, \
+        _______, _______, _______,KC_COLMK, _______, _______, _______,     _______,_______, _______, _______, _______, _______, _______, \
+                                          _______,_______, _______,       _______,              _______,      _______, _______, _______ \
```
Comment (fauxpark) [inline keyboards/keyprez/bison/keymaps/default_6_6/keymap.c:185]:
```suggestion
    [_FN] = LAYOUT_6_6(
        _______, _______, _______, _______, _______, _______,                      _______, _______, _______, _______, _______, _______,
        _______,KC_QWRTY, _______, _______, _______, _______,                      _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______,                       KC_HRM, _______, _______, _______, _______, _______,
        _______, _______, _______,KC_COLMK, _______, _______, _______,     _______,_______, _______, _______, _______, _______, _______,
                                          _______,_______, _______,       _______,              _______,      _______, _______, _______
```

File: keyboards/keyprez/bison/readme.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
+# bison
+
+![Imgur](https://i.imgur.com/FTOUHTq.jpg)
```
Comment (fauxpark) [inline keyboards/keyprez/bison/readme.md:3]:
```suggestion
![Imgur](https://i.imgur.com/FTOUHTqh.jpg)
```

File: keyboards/keyprez/bison/readme.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
+A split 60% ergonomic keyboard with a slight columnar stagger. Should provide an easy transition from a ortholinear keyboard to a split ergonomic keyboard.
+
+* Keyboard Maintainer: [csandven](https://github.com/csandven)
+* Hardware Supported: *Pro micro compatible boards*
```
Comment (fauxpark) [inline keyboards/keyprez/bison/readme.md:8]:
```suggestion
* Hardware Supported: Pro micro compatible boards
```

File: keyboards/keyprez/bison/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,226 @@
...
+    * |------+------+------+------+------+------+------+------|                    |------+------+------+------+------+------+------+------|
+    * |  F5  |  F6  | ESC  |   A  |   S  |   D  |   F  |   G  |-------.    ,-------|   H  |   J  |   K  |   L  |   ;  |  '   |  BS  |  Fn  |
+    * |------+------+------+------+------+------+------+------|       |    |       |------+------+------+------+------+------+------+------|
+    * |  F7  |  F8  |LShift|   Z  |   X  |   C  |   V  |   B  |       |    |       |   N  |   M  |   ,  |   .  |   /  | Enter| Home | End  |
+    * `-------------------------------------------------------/-------/     \------\-------------------------------------------------------'
+    *                          | LGUI | LAlt | LCTR |LOWER |  BS     /       \Space    |RAISE | RCTR | RAlt | RGUI |
+    *                          |      |      |      |      |        /         \        |      |      |      |      |
+    *                           `----------------------------------'           '------''---------------------------'
+    */
+    [_QWERTY] = LAYOUT( \
```
Comment (fauxpark) [inline keyboards/keyprez/bison/keymaps/default/keymap.c:65]: etc.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**